File: Iniciante/1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main() {
    double a, b, c, delta;
    cin >> a >> b >> c;
    std::cout << std::fixed << std::setprecision(5);
    
    if(a != 0) {
        delta = (b*b) - (4 * a * c);
        
        if(delta <= 0)
            cout << "Impossivel calcular" << endl;
        
        else {
            cout << "R1 = " << ((-b + sqrt(delta))/(2*a))  << endl;
            cout << "R2 = " << ((-b - sqrt(delta))/(2*a)) << endl;
        }
    }
    
    else 
        cout << "Impossivel calcular" << endl;
    return 0;
}

File: Iniciante/C#/1036.cs
using System;
class URI {
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        double a = double.Parse(input[0]);
        double b = double.Parse(input[1]);
        double c = double.Parse(input[2]);
        double d;
        
        if(a != 0) {
            d = (b*b) - (4 * a * c);
            if(d <= 0)
                Console.WriteLine("Impossivel calcular");
                
            else {
                Console.WriteLine("R1 = " + $"{((-b + Math.Sqrt(d))/(2*a)):0.00000}");
                Console.WriteLine("R2 = " + $"{((-b - Math.Sqrt(d))/(2*a)):0.00000}");
            }
        }
        
        else 
            Console.WriteLine("Impossivel calcular");
    }
}